da/celestia/celestia.go

```diff
@@ -81,7 +84,19 @@ func (c *DataAvailabilityLayerClient) SubmitBlock(ctx context.Context, block *ty
 		}
 	}
 
-	txResponse, err := c.client.SubmitPFB(ctx, c.namespaceID, blob, c.config.Fee, c.config.GasLimit)
+	blobs := []*blob.Blob{
+		{
+			Blob: tmproto.Blob{
```

Comment:
This requires https://github.com/rollkit/cometbft/pull/1 to be merged and the `go mod` version updated.